app/cards/__tests__/Card-test.tsx

```diff
@@ -1,7 +1,12 @@
 jest.mock('react-apollo');
+const share = jest.fn();
+Object.defineProperty(window, 'addthis', {
```

Comment:
This is fine here, but you might want to consider putting this into a [`setupFiles`](https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array) or [`setupTestFrameworkScriptFile`](https://facebook.github.io/jest/docs/en/configuration.html#setupfiles-array) so that it carries across all test environments.